CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # Changelog
 
+## 17.2.0 [#570](https://github.com/openfisca/openfisca-core/pull/570)
```

Comment:
Why is it a breaking change ? It seams it only add new possibilities

Reply:
It's not. A breaking change would be `18.0.0`. `17.2.0` means a new feature.